RealmSwift/LinkingObjects.swift

```diff
@@ -469,7 +509,7 @@ public class LinkingObjectsBase: NSObject, NSFastEnumeration {
     internal let objectClassName: String
     internal let propertyName: String
 
-    private var cachedRLMResults: RLMResults?
+    fileprivate var cachedRLMResults: RLMResults?
```

Comment:
@jpsim Yes, that's what I meant. I meant "renamed" from Swift 2 to Swift 3, not that they changed it in Swift 2.